packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -637,7 +637,7 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     final Set<LogicalKeyboardKey> keysPressed = LogicalKeyboardKey.collapseSynonyms(RawKeyboard.instance.keysPressed);
     final LogicalKeyboardKey key = keyEvent.logicalKey;
 
-    final bool isMacOS = keyEvent.data is RawKeyEventDataMacOs;
+    final bool isMacOS = defaultTargetPlatform == TargetPlatform.macOS;
```

Comment:
I wanted to be able to override platform on the unit tests, so I changed this. Please let me know if theres a better way.